packages/wallet/src/redux/protocols/advance-channel/actions.ts

```diff
@@ -18,7 +18,7 @@ export const clearedToSend: ActionConstructor<ClearedToSend> = p => {
 
 export function isAdvanceChannelAction(action: WalletAction): action is AdvanceChannelAction {
   return (
-    action.type === 'WALLET.ADVANCE_CHANNEL.COMMITMENTS_RECEIVED' ||
+    action.type === 'WALLET.COMMON.COMMITMENTS_RECEIVED' ||
```

Comment:
Now that we're making use of the `ProtocolLocator` we can add a check for `ADVANCE_CHANNEL_PROTOCOL_LOCATOR`